ide/xml/nbproject/project.xml

```diff
@@ -53,20 +53,20 @@
                     </run-dependency>
                 </dependency>
                 <dependency>
-                    <code-name-base>org.netbeans.modules.editor.lib</code-name-base>
+                    <code-name-base>org.netbeans.modules.editor.document</code-name-base>
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <release-version>3</release-version>
-                        <specification-version>4.0</specification-version>
+                        <specification-version>1.0</specification-version>
                     </run-dependency>
```

Comment:
nitpick: i believe the changes to the dependencies here are due to NB sorting the deps differently now, they can be likely reverted if all you wanted is to add the `friend-package` registration on the tail end of the file

Reply:
Well I just added this via the UI. I can revert this.